Makefile

```diff
@@ -1,7 +1,38 @@
+PROTO_VERSION=master
+PROTO_URL=https://bitbucket.org/nspcc-dev/neofs-proto/get/$(PROTO_VERSION).tar.gz
+
 B=\033[0;1m
 G=\033[0;92m
 R=\033[0m
 
+.PHONY: deps format docgen protoc
+
+# Dependencies
+deps:
+	@echo "${B}${G}=> Golang modules ${R}"
+	@go mod tidy -v
+	@go mod vendor
+
+	@echo "${B}${G}=> Cleanup old files ${R}"
+	@for f in `find . -type f -name '*.pb.go' -not -path './vendor/*' `; do \
+		rm $$f; \
```

Comment:
It is probably better to provide `-exec rm {} \;` argument to find than use for loop.

Reply:
kk